src/diffpy/structure/parsers/p_cif.py

```diff
@@ -281,8 +286,11 @@ def parseFile(self, filename):
         """
         self.ciffile = None
         self.filename = filename
-        fileurl = _quoteLocalPath(filename)
-        rv = self._parseCifDataSource(fileurl)
+        if has_pathlib:
+            rv = self._parseCifDataSource(filename)
+        else:
+            fileurl = _quoteLocalPath(filename)
+            rv = self._parseCifDataSource(fileurl)
```

Comment:
Can you verify the version of PyCifRW that works without `_quoteLocalPath` is available for Python 2.7?

If so, please specify it as a minimum required version in setup.py.
Then let's remove `_quoteLocalPath` and always use `rv = self._parseCifDataSource(filename)`.

Reply:
Yes `PyCifRW 4.4.3` is available for Python 2.7 and I have bumped the minimum version. The call to `_quoteLocalPath` has been removed and the tests pass for both Python 2.7 and Python 3 using this version.